src/main/java/gov/nasa/pds/tools/validate/BundleManager.java

```diff
@@ -504,10 +551,56 @@ public static void makeException(URL url, String location) {
         //     2.  Create a list of other bundle files to ignore so only the provided bundle is processed.
         //     3.  Create a list of collection files to ignore so only the latest collection file is processed.
 
+        LOG.debug("makeException:url,location {},{}",url,location);
+
         // First, find any other bundle files so they can be eliminated from crawling later.
         ArrayList<Target> otherBundleFiles = BundleManager.findOtherBundleFiles(url);
         BundleManager.m_ignoreList.addAll(otherBundleFiles);
 
+        // Because this function has knowledge of why a particular file is being ignored while crawling
+        // we will log it here.
+        if (otherBundleFiles != null) {
+            // Only print the size if we know that otherBundleFiles is not null.
+            LOG.debug("makeException:url,otherBundleFiles.size() {},{}",url,otherBundleFiles.size());
+            for (Target target : otherBundleFiles) {
+                LOG.info("SKIP: {} due to not being selected as the bundle target",target.getUrl());
+                    // Write a record to report that we are skipping this file.
+                    if (BundleManager.m_report != null) {
+                        try {
+                            // Reminder for enumerated values of ExceptionType:
+                            // public enum ExceptionType {
+                            //   DEBUG(5,"DEBUG"),
+                            //   INFO(4,"INFO"),
+                            //   WARNING(3,"WARNING"),
+                            //   ERROR(2,"ERROR"),
+                            //   FATAL(1,"FATAL_ERROR");
+                            // In our case, we want ExceptionType to be WARNING so it gets printed in the report for default verbose level of 3 (WARNING)
+                            ValidationProblem p1 = null;
+                            if ((FlagsUtil.getSeverity() != null) &&
```

Comment:
@qchaupds i think we may be over-complicating this a bit? I do not think we need any custom logic for handling things based upon the severity set in the parent or anything like that. apologies if that was not clear in my previous example. I think this should just be:

```
                                p1 = new ValidationProblem(new ProblemDefinition(ExceptionType.INFO,
                                                               ProblemType.UNREFERENCED_FILE, "Skipping " + target.getUrl() + " due to not being selected as the bundle target"),
                                                           target.getUrl());
```

no if/then logic necessary based upon severity. all of that is handled when this is reported out.